### DISCUSSION TITLE
When will rdkit be available on Python 3.13?

### DISCUSSION THREAD

**Discussion by CNSeniorious000** [rdkit/rdkit#8103] [Q&A] [answered]
Python 3.13 offers a free-threaded feature, which I believe is beneficial for CPU-intensive tasks. But it seems rdkit still don't offers a 3.13 wheel now.

**Comment by greglandrum** [comment, marked as answer]
The RDKit conda builds are available for Python 3.13 already

**Reply by CNSeniorious000** [reply]
Thanks! Do you have a plan to publish 3.13 wheels to PyPI too?

**Reply by greglandrum** [reply]
> Thanks! Do you have a plan to publish 3.13 wheels to PyPI too?

I don't do the PyPi builds (there's ample material on the web that explains why). If you have questions about those, ask in the rdkit-pypi repo.

**Reply by CNSeniorious000** [reply]
Thanks! I didn't notice that the rdkit on PyPI is actually this [rdkit-pypi](https://github.com/kuelumbus/rdkit-pypi) repo.

**Comment by anmorgunov** [comment]
Just to save someone else his time -- there seem to be no builds for free threaded python even in conda forge.
